shop-service/product-service/test/getProductsById.test.js

```diff
@@ -0,0 +1,46 @@
+import { getProductsById } from "../handlers/getProductsById";
+import mockData from "../mockData";
+
+describe("product service", () => {
+  let event;
+
+  beforeEach(() => {
+    event = {
+      pathParameters: {
+        id: mockData[0].id,
+      },
+    };
+  });
+
+  it("return well-formed response", () => {
```

Comment:
I believe jest supports async/await to not use `.then` inside the specs

Reply:
done